PrBuild/index.js

```diff
@@ -1,5 +1,5 @@
-var request = require('request-promise');
-var _ = require('lodash');
```

Comment:
Can we drop this from package.json as well? I intended to use it, but there is no need for lodash right now.

Reply:
Removed, thanks.